spec/cypress/integration/Nominations.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Resend nominations flow
+  All users need to be able to attempt to resend nominations for any school
+
+  Scenario: Resending nomination email
+    Given scenario "school_with_local_authority" has been run
+    And I am on "resend nominations choose location" page
+    Then the page should be accessible
+
+
+    When I type "test" into accessible autocomplete
+    And I click on the first element in the displayed list
```

Comment:
Have you seen how I'm handling this in the lead provider user management PR? I don't think it needs any new steps: https://github.com/DFE-Digital/early-careers-framework/pull/179/files#diff-4091c82e65b1ea35716953ce7a2b31cf418ffc45c1961678631e84307f39ce10R14

Reply:
Ah, great, two PRs in parallel figured out different ways of cucumbering accessible autocomplete. Fine, merge your changes and I will reuse them here.

Reply:
🙇done!

Reply:
and done!